abstracts/chubby.md

```diff
@@ -1,3 +1,12 @@
+We describe our experiences with the Chubby lock service, which is intended to
```

Comment:
It looks like this block duplicates the one below, but with your "and" -> & change. Why not remove the extra block and do the & replacement below?

Reply:
We can do this in a follow-up PR